src/kimmdy/runmanager.py

```diff
@@ -398,6 +398,18 @@ def _setup(self, files: TaskFiles) -> TaskFiles:
         write_top(self.top.to_dict(), files.outputdir / self.config.top.name)
         files.output["top"] = files.outputdir / self.config.top.name
         logger.info("Done with setup")
+
+        # FIXME: find a better solution for this
+        #
+        # copy input files that are potentially modified
+        # to the setup task directory
+        # by applying a recipe (e.g. by trunkate_sim_files)
+        # for f in ["xtc", "tpr", "trr"]:
```

Comment:
Maybe we should have this code here in place until we improve it? At least it works, even if it comes with a performance hit in some cases.

Reply:
I suppose it also only has the performance hit in cases where you actually have an initial trajectory, which are also exactly the cases in which you don't want it to be overwritten. Not deleting user data seems to be the easier choice for the tradeoff. Will add back in.